### DISCUSSION TITLE
Limitations of model input

### DISCUSSION THREAD

**Discussion by havetry** [PaddlePaddle/PaddleSpatial#12] [BaiduKDDCup2022] [answered]
Dear organizers：
   When predicting the Patv after time-t, can we use the Wspd after time-t as the forecast meteorological data? At the same time, can Wdir, pab1 and other data after time t be used as input?

**Comment by zhoujingbo** [comment, marked as answer]
Dear havetry;

The data of Wspd, Wdir and pab1 after time-t cannot be used. In our settings, we aim to forecast the power generated by a wind farm with the Supervisory Control and Data Acquisition (SCADA) data and Spatial data on top of the spatiotemporal modeling paradigm without knowing the future meteorological data. 

Thanks a lot for your support.

Regards

**Reply by zlj-cs** [reply]
does it mean that we cannot foresee any information except TurbID, Day, and Tmstamp?

**Reply by zhoujingbo** [reply]
Yes, it does.
